models/payment.py

```diff
@@ -32,18 +33,18 @@ def _get_nelo_urls(self):
         if self.state == 'enabled': #prod
             return {
                 'web_url': self._nelo_redirect_url,
-                'rest_url': 'https://api-v2.nelo.co/v1'
+                'rest_url': 'https://api.nelo.co/v1'
             }
         else:
             return {
                 'web_url': self._nelo_redirect_url,
-                'rest_url': 'https://api-v2-dev.nelo.co/v1'
+                'rest_url': 'https://dev.nelo.co/v1'
```

Comment:
ah sorry, this should actually be `sandbox.nelo.co/v1`

dev is only for internal nelo